compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParser.java

```diff
@@ -1669,9 +1669,8 @@ private STNode getUpdatedParamList(STNode parameters, int index) {
                 case REQUIRED_PARAM:
                     STRequiredParameterNode requiredParam = (STRequiredParameterNode) param;
                     if (isEmpty(requiredParam.paramName)) {
-                        param = STNodeFactory
-                                .createRequiredParameterNode(requiredParam.annotations, requiredParam.typeName,
-                                        paramName);
+                        param = STNodeFactory.createRequiredParameterNode(SyntaxKind.REQUIRED_PARAM,
+                                requiredParam.annotations, null, requiredParam.typeName, paramName);
```

Comment:
Can we use `STNodeFactory.createEmptyNode()` instead of `null`?

Reply:
Fixed.